### DISCUSSION TITLE
Reason for declining a request

### DISCUSSION THREAD

**Discussion by jeremysherriff** [sct/overseerr#2270] [Feature Requests]
Hey all.  I had a quick search but didn't see any previous discussions on the topic.

It would be great to be able to supply a reason when declining a request, I'd be happy with either a free-form field I can type something in, or bunch of pre-canned responses ("Available on Netflix", "Available on a streaming service", "Had it before and it's crap" or whatever)

What's people's thoughts? is this something you'd use?

Edit: looks like it's already a request in the Issues section ( #593 )

**Comment by TheCatLady** [comment]
Discussing the feature request here is fine, but please do not add "+1"-type comments to issues in the future.  We encourage using the 👍 reaction in lieu of such comments, so as to not spam everyone subscribed to the issue.

**Reply by shark2k** [reply]
Any chance this feature request has been decided to be implemented?  It would be nice to be able to add a comment if declining something (I try to do that very frequently if I can but sometimes it needs to be done).  Since a notification can already go to the user letting them know it's declined, it would be nice to have the reason in there so that we wouldn't have to reach out separately to explain.

Thanks,
-Shark2

**Comment by Magic815** [comment]
Having put some thought into this myself, my suggestion would be to have a shortlist of pre-canned responses (which can be edited in Overseerr settings to each users' liking), but also give the admin user the opportunity to type a one-off custom text at the time of rejecting - if none of the pre-canned options fit the bill.

Also, since Overseerr already knows what movies and shows are currently streaming on (per the media info box):
![image](https://user-images.githubusercontent.com/9938050/143264410-6278dd60-7002-4e39-8fdc-0006606f7699.png)

It would be really cool if the pre-canned "Currently streaming on" reject option would just automatically mention that streaming service by name - without the admin having to manually select the right streaming service name out of a list. So in that example above, "Currently streaming on" would become "Currently streaming on Amazon Prime Video." That way there wouldn't need to be a dozen pre-canned list options, for each streaming service name.

Other pre-canned responses that I think could work decently well as Overseerr defaults:
"Too niche / no download sources are available" (for when something has already come out but is not available on the admins torrent/usenet sources)
"No Reality TV allowed"
"Not the type of content I want to store"

**Comment by syaoran68** [comment]
I'm gonna revive this from the grave.... I'd definitely like this since i don't want to keep requests sometimes in my backlog.. but also don't want to seem like a heartless A-hole that just declines peoples request.. 😂

**Comment by 3rdStng** [comment]
Came looking for this exact feature. It is the one and only thing I miss from Ombi. The responses don't even need to be pre-canned.  A simple text input box for the reason is enough. The rest of the notification can be a template.

**Comment by mroach** [comment]
I also came looking for this. My current workflow is amending or declining a request and then texting the person telling them why so they don't think I'm being unreasonable. A common problem with my users is they'll want to check out a show they heard of, and they do this by requesting every single season of it. I don't think they're necessarily doing that intentionally, just going with defaults. I'd like to be able to edit their request and only fetch the first season and include a note like "If you like it, feel free to request more"

Sometimes the answers are more specific like "this is on youtube" or "I don't want every Jackass movie in my library"

Maybe this would work well as generic comments on requests? That would also allow the requester to leave a comment when making a request and/or reply to admin comments.

**Comment by samohtxotom** [comment]
check out PR #4205 hopefully coming to the develop branch soon!
